docs/cloud-adoption/operations/operational-fitness-review.md

```diff
@@ -0,0 +1,127 @@
+
+### Process
+
+The operational fitness review team meets on a regular basis. The team meets on a monthly cadence and reports status and metrics to senior leadership on a quarterly basis.
+
+Each time the team meets, they work through the following tasks:
+
+1. The business owner and business advocate enumerate and determine the non-functional requirements for each of business operation with input from the engineering and service owners. For business operations that have been previously identified, the priority is reviewed and verified. For new business operations, a priority in the existing list is assigned.
+
+2. The engineering and service owners map the **current state** of business operations to the corresponding on-premises and cloud services. The mapping is composed of a list of the constituent components in each service oriented as a dependency tree. Once the list and dependency tree is generated, the **critical paths** through the tree are determined.
+
+3. The engineering and service owners review the current state of operational logging and monitoring for the services listed in the previous step. Robust logging and monitoring are critical to identifying service components that are contributing to failures in meeting non-functional requirements. If sufficient logging and monitoring are not in place, a plan must be created and implemented to put it in place.
+
+4. Scorecard metrics are created for new business operations. The scorecard is composed of the list of constituent components for each service identified in step 2 aligned with the non-functional requirements and a metric representing the how satisfactorily the component meets the requirement.
+
+5. Next, for those constituent components that fail to meet non-functional requirements, a high-level solution is designed and an engineering owner is assigned. At this point the business owner and business advocate should establish a budget for the remediation work based on the expected revenue of the business operation.
+
+6. Finally, a review is conducted of the ongoing remediation work. Each of the scorecard metrics for work that is in flight is reviewed against the expected metrics. The service owner for those constituent components that are meeting metrics present logging and monitoring data to confirm that the metric is met. For those constituent components that are not meeting metrics, each engineering owner explains the issues that are preventing metrics from being reached and any new designs for remediation.
```

Comment:
> expected metrics

Maybe I'm nit-picking but I think of a metric as the value you're measuring, versus the target that you're trying to hit. You meet or miss the target, not the metric.